README.md

```diff
@@ -1,10 +1,14 @@
 # Sublime Packages
 
-> **These packages are developed against the latest [Sublime Text Dev Build](http://sublimetext.com/dev). Bugs may exist on older builds, and the format used is not compatible with builds older than 4095.**
+## Requirements
+
+These packages are developed against the latest Build of **Sublime Text** in the [`dev` channel](https://www.sublimetext.com/dev).
```

Comment:
Could also mention that the state of the repo for specific shipped ST builds can be seen by browsing tags: https://github.com/sublimehq/Packages/tags

Reply:
Addressed in sublimehq/Packages@bee3474.